charts/activator-filter/templates/istio.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: {{ .Values.name }}-ext-proc
+  namespace: {{ .Release.Namespace }}
+spec:
+  configPatches:
+  - applyTo: HTTP_FILTER
+    match:
+      # context omitted so that this applies to both sidecars and gateways
+      listener:
+        filterChain:
+          filter:
+            name: "envoy.filters.network.http_connection_manager"
+    patch:
+      operation: INSERT_FIRST # TODO: insert before EPP
```

Comment:
OOC: what will happen if this is deployed together with BBR - which ext proc will be invoked first?

Reply:
it depends on the order of which filters are applied. It really does not matter which one run first, maybe more optimal to have BBR before the activator as the BBR can reject invalid requests.

Reply:
exactly. you are touching exactly the point I was trying to highlight.
It might be worth adding this information so some section in the deployment instructions.
we should recommend in which order to deploy them (and I agree as you mentioned that BBR should run first).

Reply:
I added a note in the readme.